packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -83,7 +82,9 @@ describe("ActionMenu", () => {
         expect(onClick).toHaveBeenCalledTimes(1);
 
         // Have to reopen menu because menu closes after an item is selected
-        menu.setState({open: true});
+        menu.find(ClickableBehavior)
+            .at(0)
+            .simulate("click");
```

Comment:
There should only be one so `find(...).first().simulate("click")` should also work.  `.at(0)` is the same as `.first()` so this is fine as well.

Reply:
I’ll change it to first(), looks a little clearer to me